package.json

```diff
@@ -112,7 +112,7 @@
     "connect-flash": "^0.1.1",
     "connect-redis": "^7.1.1",
     "csurf": "^1.11.0",
-    "date-fns": "^2.30.0",
+    "date-fns": "^3.4.0",
```

Comment:
this is a bit of a [scary jump](https://github.com/date-fns/date-fns/blob/main/CHANGELOG.md)!

Reply:
😱